docs/component_conventions.md

```diff
@@ -201,3 +201,73 @@ Code can be called and referred to in the template as follows:
 
 <%= card_helper.heading_tag %>
 ```
+
+## Passing HTML to a component
+To avoid the risk of Cross Site Scripting (XSS) attack, we only mark HTML as safe  only sanitise HTML when the component is used, rather than doing so on the application's behalf.
```

Comment:
I think this is missing a word or some punctuation? `we only mark HTML as safe  only sanitise HTML`

Reply:
Ah yeah, that's not right, thanks Andy.